### PR TITLE
fix: 7mode dashboard

### LINKED ISSUE #383: NetApp 7mode dashboard request
A customer is having trouble seeing certain information on the 7mode dashboard. Below are three examples of what we would like adjusted on the dashboard and each example is accompanied by a screenshot that is in the word doc attached.

Screenshot 1:  
Left: CMODE
Right:  7MODE
 
-  "Physical Space Used" does not show on 7MODE.
-  Also, "Space Used%” calculation is questionable from us when looking at the “Total Space” and "Available Space”

Screenshot 2:
 
For the "Netapp Details Volume" dashboard, metrics only show on CMODE, not 7MODE

Screenshot 3:
If we have the same dashboard for 7mode and CMODE, how is it going to work since 7mode does not have SVM variable?  Although we tried to remove the SVM variable to test the 7mode, it is still not working.

harvestdash1image025

### PR DESCRIPTION
**Pull request by rahulguptajss**
fixes #383 #415

### CODE REVIEW

**Comment by rahulguptajss** [comment]
Pending 7mode issues.

**Cluster:**
health : missing information in zapi
disk busy: aggregator.go doesnt process global labels. Below warning
{"level":"warn","Poller":"7mode2","plugin":"ZapiPerf:Aggregator","caller":"goharvest2/cmd/poller/plugin/aggregator/aggregator.go:154","time":"2021-08-18T20:52:09+05:30","message":"label name for [node] missing, skipped"}

**disk:**
Missing "flash cache" and "flash pool" in dashboard

**Network:**
Missing FCP Send, FCP Receive, Transmission interrupts, Transmission errors, Link Down, Link Failure

**Network with Nvme/FC**
Missing FCP Send, FCP Receive, Nvme/FC Receive, Nvme/FC send, Transmission errors, Link Down, Link Failure
Need full dshboard check

**Aggregate:** Missing Mappings in 7 mode. Possible that 7 mode do not support these
1. Physical Space Used      
2. Capacity Tier Used           
3. Data Compaction space used   
4. space used for snapshot %  

**fcpport:**
{"level":"error","Poller":"7mode2","collector":"ZapiPerf:FcpPort","stack":[{"func":"New","line":"35","source":"errors.go"},{"func":"(*Fcp).Run","line":"29","source":"fcp.go"},{"func":"(*AbstractCollector).Start","line":"329","source":"collector.go"},{"func":"goexit","line":"1371","source":"asm_amd64.s"}],"error":"no metrics => read_data","caller":"goharvest2/cmd/poller/collector/collector.go:330","time":"2021-08-18T20:51:55+05:30","message":"plugin [Fcp]: "}

**Review by cgrinds — APPROVED** [review]
nice job @rahulguptajss

File: conf/zapi/7mode/8.6.0/disk.yaml [hunk 1]
```diff
@@ -23,6 +23,8 @@ counters:
 plugins:
   - LabelAgent:
     value_mapping: status offline false `0`
+    # metric label zapi_value rest_value `default_value`
+    value_to_num: new_status offline - - `0` #ok_value is empty value, '-' would be converted to blank while processing.
```
Comment (Hardikl) [inline conf/zapi/7mode/8.6.0/disk.yaml:27]: value of offline field would be empty or boolean ?

File: conf/zapiperf/7mode/8.2.5/volume_node.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 
 name:                     VolumeNode
 query:                    volume
-object:                   volume
-
+object:                   node_vol
```
Comment (Hardikl) [inline conf/zapiperf/7mode/8.2.5/volume_node.yaml:4]: where is node_vol used?

File: conf/zapi/7mode/8.6.0/disk.yaml (hunk 1 above)
Comment (rahulguptajss) [reply conf/zapi/7mode/8.6.0/disk.yaml:27]: doh, we don't have is-offline in 7mode disk response. Added to known issues in https://github.com/NetApp/harvest/issues/423

File: conf/zapiperf/7mode/8.2.5/volume_node.yaml (hunk 2 above)
Comment (rahulguptajss) [reply conf/zapiperf/7mode/8.2.5/volume_node.yaml:4]: no where yet.  This template is not utilized in current dashboards

**Review by Hardikl — APPROVED** [review]

**Pull request merged**